building/build-debs/homeworld-admin-tools/inner-build.sh

```diff
@@ -3,6 +3,21 @@ set -e -u
 
 rm -rf spire spire.zip src/resources src/__pycache__ resources/__pycache__
 cp -RT resources src/resources
+
+if [[ -v VERSION ]]
+then
+    echo "$VERSION" >src/resources/DEB_VERSION
+else
+    echo "Spire was not built as part of a Debian package." >src/resources/DEB_VERSION
+fi
+
+if [[ ! -e src/resources/GIT_VERSION ]]
+then
+    git rev-parse HEAD | tr -d '\n' >src/resources/GIT_VERSION
+    [[ -n "$(git status --porcelain)" ]] && echo -n "-dirty" >>src/resources/GIT_VERSION
+    echo >>src/resources/GIT_VERSION
+fi
```

Comment:
I assume that the reason that this code is duplicated is because the git repository can't be accessed from within the build chroot?

In any case, you should make sure that inner_build.sh cleans up GIT_VERSION.

Reply:
I am writing it to src/resources/GIT_VERSION, which should already be cleaned up by the existing code. Though this is not done with a trap. Should I change it to use a trap?

Reply:
Woops, you're right. It's good as it is.